.script/tests/KqlvalidationsTests/Kqlvalidations.Tests.csproj

```diff
@@ -5,13 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Sentinel.Analytics.Management.AnalyticsManagement.Contracts" Version="11.7.2" />
```

Comment:
not sure we want it here - maybe have a copy of the data model - let's talk about it